src/commissaire_http/dispatcher/__init__.py

```diff
@@ -210,89 +151,38 @@ def dispatch(self, environ, start_response):
             raise DispatcherError(
                 'Bus can not be None when dispatching. '
                 'Please call dispatcher.setup_bus().')
-        route_info = self._router.routematch(environ['PATH_INFO'], environ)
-
-        # If we have valid route_info
-        if route_info:
-            # Split up the route from the route data
-            route, route_data = route_info
-
-            # Get the parameters
-            try:
-                params = self._get_params(environ, route, route_data)
-            except Exception as error:
-                start_response(
-                    '400 Bad Request',
-                    [('content-type', 'text/html')])
-                return [bytes('Bad Request', 'utf8')]
-
-            # method is normally supposed to be the method to be called
-            # but we hijack it for the method that was used over HTTP
-            jsonrpc_msg = {
-                'jsonrpc': '2.0',
-                'id': str(uuid.uuid4()),
-                'method': environ['REQUEST_METHOD'],
-                'params': params,
-            }
 
+        # Add the bus instance to the WSGI environment dictionary.
+        environ['commissaire.bus'] = self._bus
```

Comment:
Yeah, that's actually mandated by [PEP 3333](https://www.python.org/dev/peps/pep-3333/#environ-variables):

Finally, the `environ` dictionary may also contain server-defined variables. These variables should be named using only lower-case letters, numbers, dots, and underscores, and should be prefixed with a name that is unique to the defining server or gateway. For example, `mod_python` might define variables with names like `mod_python.some_variable`. 
```